applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -58,12 +58,16 @@ def __init__(self, main_model_part, custom_settings):
             "volume_model_part_name" : "volume_model_part",
             "skin_parts": [""],
             "no_skin_parts":[""],
+            "time_stepping"                : {
+                "automatic_time_step" : true,
+                "CFL_number"          : 1,
+                "maximum_delta_time"  : 0.01
+            },
             "alpha":-0.3,
             "move_mesh_strategy": 0,
             "periodic": "periodic",
             "regularization_coef": 1000,
             "MoveMeshFlag": false,
-            "use_slip_conditions": false,
             "turbulence_model": "None",
             "use_spalart_allmaras": false
```

Comment:
If we remove the slip_conditions, the turbulence_model and spalart_allmaras flags should go too (they have a very limited support at the moment, I would hide them from the interface). The same might apply for the regularization_coef.